src/toolkits/activity_classification/ac_data_iterator.cpp

```diff
@@ -296,6 +296,12 @@ variant_map_type _activity_classifier_prepare_data_aug_impl(
   }
 #endif
 
+  for (auto &feat : features) {
```

Comment:
It's probably best not to use `auto` here.

Reply:
yeah, flexible_type is enough.

Reply:
sorry, std::string.

Reply:
very good!